tests/core/test_data.py

```diff
@@ -105,19 +105,6 @@ def test_same_file_names_get_resolved(tmpdir):
     assert all([f.endswith("stories.md") for f in stories])
```

Comment:
can you please also add a test which checks whether `_find_core_nlu_files_in_directory` detects luis, wit files?

Reply:
Yes, the new test is [here](https://github.com/RasaHQ/rasa/blob/39e540c1c15aa6f323d6e722a32285394778ed6e/tests/core/test_data.py#L109)